_includes/pbc/all/upgrade-modules/upgrade-the-country-module.md

```diff
@@ -0,0 +1,22 @@
+This document describes how to upgrade the Country module.
+
+## Upgrading from version 3.* to version 4.0.0
+
+In this new version of the `Country` module, we have added support configuration currency for each store in database.
+With the `Country` module version 4 we have added the `spy_country_store` database table to persist stores-countries in Zed.
+
+You can find more details about the changes on the [Country module](https://github.com/spryker/country/releases) release page.
+
+*Estimated migration time: 5 min*
+
+To upgrade to the new version of the module, do the following:
+
+1. Upgrade the `Country` module to the new version:
+
+```bash
+composer require spryker/country:"^4.0.0" --update-with-dependencies
+```
+
```

Comment:
https://github.com/spryker/spryker/pull/9457/files?file-filters%5B%5D=.csv&file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.md&file-filters%5B%5D=.neon&file-filters%5B%5D=.php&file-filters%5B%5D=.twig&file-filters%5B%5D=.txt&file-filters%5B%5D=.xml&file-filters%5B%5D=.yaml&file-filters%5B%5D=.yml&file-filters%5B%5D=No+extension&file-filters%5B%5D=dotfile&show-viewed-files=true&show-deleted-files=false#diff-0b92f94ce075a7d30216f603b5ef8348e9749ad6f66f7da2d9b6190189f947df

This change is not mentioned but requires some action items.

Reply:
I can't open this link